### PR TITLE
front: fix decimals error for project budget

### LINKED ISSUE #9662: Selecting float budget gives unreadable error in project
### What happened?

Float budget gives error

### What did you expect to happen?

I expect the field to be highlighted in red, with a clear message stating that float numbers are forbidden

### How can we reproduce it (as minimally and precisely as possible)?

Go to operational studies page
Open project creation form
Enter the name and a floating point budget
Click on save, witness

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Firefox

### OSRD version (top right corner `Account` button > `Informations`)

f05eecf

### PR DESCRIPTION
**Pull request by Akctarus**
closes #9662

### CODE REVIEW

File: front/public/locales/en/operationalStudies/project.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   "objectives": "goals",
   "openStudy": "Open",
   "projectBudget": "Estimated budget",
-  "projectBudgetInvalid": "The budget for the project must not exceed 2147483647",
+  "projectBudgetInvalid": "The project budget must be a positive integer, and must not exceed 2147483647 €.",
```
Comment (theocrsb) [inline front/public/locales/en/operationalStudies/project.json]:
```suggestion
  "projectBudgetInvalid": "The project budget must be a positive integer and must not exceed 2147483647 €",
```

File: front/public/locales/fr/operationalStudies/project.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
   "objectives": "objectifs",
   "openStudy": "Ouvrir",
   "projectBudget": "Budget estimé",
-  "projectBudgetInvalid": "Le budget du projet ne doit pas dépasser 2147483647",
+  "projectBudgetInvalid": "Le budget du projet doit être un entier positif, et ne doit pas dépasser 2147483647 €",
```
Comment (theocrsb) [inline front/public/locales/fr/operationalStudies/project.json]:
```suggestion
  "projectBudgetInvalid": "Le budget du projet doit être un entier positif et ne doit pas dépasser 2147483647 €",
```

**Review by emersion — APPROVED** [review]
LGTM :)

**Review by theocrsb — APPROVED** [review]
LGTM and tested

**Pull request merged**